src/app/pages/component-sidenav/_component-sidenav-theme.scss

```diff
@@ -1,4 +1,4 @@
-@import '~@angular/material/theming';
+@import '../../../../node_modules/@angular/material/theming';
```

Comment:
@jelbourn  As an update on this issue, compiling the individual `scss` files for each theme doesn't have access to webpack, but instead uses `node-sass`, so the `~` operator will not work here unfortunately.